<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget_Choose</class>
 <widget class="QWidget" name="Widget_Choose">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="Group_Main">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>561</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QGroupBox{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(225, 231, 230, 245), stop:1 rgba(255, 255, 255, 255));
}</string>
   </property>
   <widget class="QLabel" name="lb_title">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_text">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>441</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cancel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_confirm">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
